index.js

```diff
@@ -33,19 +39,32 @@ function registerListener(session, opts = {}, cb = () => {}) {
 		}
 
 		item.on('updated', () => {
-			const ratio = item.getReceivedBytes() / totalBytes;
+			downloadItems[item.getStartTime()] = {
```

Comment:
I think it would be better to just use a `Set` for `downloadItems`.

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Set